_topic_map.yml

```diff
@@ -1750,13 +1745,18 @@ Topics:
 # Metering
 - Name: Using metering with OpenShift Serverless
   File: serverless-metering
-#---
-#Name: Kubernetes Cluster Federation
-#Dir: kubefed
-#Distros: openshift-enterprise
-#Topics:
-### Overview
-#- Name: Understanding Kubernetes Cluster Federation
-#  File: understanding-kubefed
-#- Name: Installing OpenShift KubeFed
-#  File: installing-kubefed
+# HA
+- Name: High availability on OpenShift Serverless
+  File: serverless-HA
+# Tracing
+- Name: Tracing requests
+  File: serverless-tracing
+# Metering
+- Name: Using metering with OpenShift Serverless
```

Comment:
It is as it says "duplicated ids". You have the serverless-metering included twice. Once here, and once above on lines 1746/47.

Reply:
idk how I didn't notice this, but thanks Vikram!